include/triton/core/tritonserver.h

```diff
@@ -2028,6 +2028,16 @@ TRITONSERVER_DECLSPEC struct TRITONSERVER_Error*
 TRITONSERVER_ServerOptionsSetModelNamespacing(
     struct TRITONSERVER_ServerOptions* options, bool enable_namespace);
 
+/// Enable peer access to allow GPU device to directly access the memory of
+/// another GPU device
+///
+/// \param options The server options object.
+/// \param enable_peerAccess Whether to enable peer access or not.
+/// \return a TRITONSERVER_Error indicating success or failure.
+TRITONSERVER_DECLSPEC struct TRITONSERVER_Error*
+TRITONSERVER_ServerOptionsSetPeerAccess(
```

Comment:
```suggestion
TRITONSERVER_ServerOptionsSetTryEnablingPeerAccess(
```
```suggestion
TRITONSERVER_ServerOptionsSetPeerAccess(
```

Reply:
Can we document that peer access may not be enabled successfully even if it is set and remove `TryEnabling` in the API?